packages/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/MarkersController.java

```diff
@@ -63,6 +63,21 @@ void removeMarkers(List<Object> markerIdsToRemove) {
     }
   }
 
+  void showMarkerInfoWindow(String markerId) {
+    MarkerController markerController = markerIdToController.get(markerId);
```

Comment:
Possibly missing a check if `markerId` exists in `markerIdToController`.
I suggest that we return an error to dart if we cannot find `markerId`

Reply:
Could do it like this, and repeat with hide and isInfoWindowShown. any thoughts on this?
```Java
  void showMarkerInfoWindow(String markerId, MethodChannel.Result result) {
    MarkerController markerController = markerIdToController.get(markerId);
    if (markerController != null) {
      markerController.showInfoWindow();
      result.success(null);
    } else {
      result.error("markerId not found", "showMarkerInfoWindow called with invalid markerId", null);
    }
  }
```

Reply:
Looks good, remember to remove the extra `result.success` after this method was called.